### PR TITLE
Remove 2 bashisms, unsure about the last.

### PR DESCRIPTION
**Pull request by kij**
removed 2/3 complains from checkbashisms, owner of the last - but unsure how to fix.

### CODE REVIEW

File: binr/r2pm/r2pm [hunk 1]
```diff
@@ -243,7 +243,7 @@ R2PM_List() {
 R2PM_Search() {
 
 	#helper for pretty printing tabbed lists
-	if ! type "column" > /dev/null; then
+	if ! command -vp column > /dev/null; then
```
Comment (radare) [inline binr/r2pm/r2pm]: i dont think type is a bashism. is it?

**Comment by Maijin** [comment]
Hey I did not know about "checkbashisms", that could be nice to add as a reference in DEVELOPPERS.md and shellcheck.sh script

**Comment by kij** [comment]
I'll look into that.

File: binr/r2pm/r2pm (hunk 1 above)
Comment (kij) [reply binr/r2pm/r2pm]: https://wiki.ubuntu.com/DashAsBinSh#type

Before
```
 kij@rohana  ~/Project/radare2  ➦ 33a01a57f  checkbashisms binr/r2pm/r2pm 
possible bashism in binr/r2pm/r2pm line 246 (type):
	if ! type "column" > /dev/null; then
possible bashism in binr/r2pm/r2pm line 249 ($'...' should be "$(printf '...')"):
		cols() { column -s $'\t' -t; }
possible bashism in binr/r2pm/r2pm line 338 (unsafe echo with backslash):
	echo "\033[31mERROR: \033[32m$@\033[0m"
```
After:
```
 kij@rohana  ~/Project/radare2   master  checkbashisms binr/r2pm/r2pm
possible bashism in binr/r2pm/r2pm line 250 ($'...' should be "$(printf '...')"):
		cols() { column -s $'\t' -t; }
```

Comment (radare) [reply binr/r2pm/r2pm]: i see som scary inconsistencies in the behaviour of `command -vp`

screen shot 2018-08-23 at 01 26 00

**Comment by radare** [comment]
the printf change is good. the command -tv one im not sure at all. and iirc type works everywhere

File: binr/r2pm/r2pm (hunk 1 above)
Comment (kij) [reply binr/r2pm/r2pm]: That is expected due to the -p,  my mistake. -p should not be there.  I'll resubmit without  'column'

**Pull request merged**